scripts/db_migrator.py

```diff
@@ -567,6 +567,25 @@ def migrate_telemetry(self):
         if not certs:
             self.configDB.set_entry("TELEMETRY", "certs", telemetry_data.get("certs"))
 
+    def migrate_gnmi(self):
+        # GNMI - add missing key
+        if not self.minigraph_data or 'GNMI' not in self.minigraph_data:
+            # If there's no GNMI in minigraph, copy configuration from CONFIG_DB TELEMETRY table
+            gnmi = self.configDB.get_entry('TELEMETRY', 'gnmi')
```

Comment:
>get_entry [](http://example.com/codeflow?start=33&length=9)

Will this line still work if `TELEMETRY` table is completely missing in ConfigDB?

Reply:
yes, if TELEMETRY table is missing, db_migrator will not generate GNMI table.